hardhat.config.ts

```diff
@@ -51,6 +51,7 @@ const config: HardhatUserConfig = {
   zksolc: {
     version: "1.5.1",
     settings: {
+      isSystem: true,
```

Comment:
what is this flag?

Reply:
This is to enables Yul instructions available only for ZKsync system contracts. We need this flag to be able to deploy contracts from contract. I checked with ML before and there is no concern enabling this flag.

Reply:
I see now